### DISCUSSION TITLE
如何Token.attrPush()支持更多的属性？

### DISCUSSION THREAD

**Discussion by JalenChung** [sbfkcel/towxml#292] [Q&A] [answered]
如果我想让Token.attrPush()支持更多的属性我该怎么做呢？查看config.js目前只有'class', 'data', 'id','style',。如果我想让它支持“value”呢？
![image](https://github.com/user-attachments/assets/f474c7b5-dbe6-4005-99ad-a076b66da7ad)

**Comment by sbfkcel** [comment, marked as answer]
是的，添加对应的属性即可。然后重新构建 towxml
